File: code-examples/cpp/usage-examples/changeStream.cpp
```cpp
// Libraries to include
#include <bsoncxx/json.hpp>
#include <mongocxx/client.hpp>
#include <mongocxx/instance.hpp>
#include <mongocxx/pipeline.hpp>
#include <mongocxx/change_stream.hpp>

int main() {
    // Initialize MongoDB instance
    mongocxx::instance inst{};
    
    // Replace the uri string with your MongoDB deployment's connection string.
    mongocxx::client client{mongocxx::uri{"<connection string uri>"}};

    // Connect to the "insertDB" database
    auto database = client.database("insertDB");
    
    // Connect to the "haikus" collection
    auto haikus = database.collection("haikus");

    // Create pipeline and set options
    mongocxx::options::change_stream opts{};
    mongocxx::pipeline pipeline{};

    // Open a Change Stream on the "haikus" collection and print changes
    auto change_stream = haikus.watch(pipeline.view(), opts);
    for (auto&& change : change_stream) {
        std::cout << "Received change:\n" << bsoncxx::to_json(change) << "\n";
    }
    
    // No explicit closing of change streams or connections needed as it is handled by the MongoDB C++ Driver

    return 0;
}
```

File: code-examples/php/usage-examples/changeStream.php
<?php
require 'vendor/autoload.php'; // Include the MongoDB PHP driver

// Replace the uri string with your MongoDB deployment's connection string
$uri = "<connection string uri>";

// Initialize the MongoDB client
$client = new MongoDB\Client($uri);

// Get the 'insertDB' database and 'haikus' collection
$database = $client->selectDatabase('insertDB');
$haikus = $database->selectCollection('haikus');

// Function to manage the change stream
function watchChanges($haikus){
    // Initialize the change stream
    $changeStream = $haikus->watch();

    try {
        // Iterate over all change events and print them out
        foreach ($changeStream as $change) {
            echo "Received change: \n";
            print_r($change);
        }
    } finally {
        // No matter what, release the resources by closing the change stream and client connection
        $changeStream->close();
        $client->close();
    }
}

// Start watching for changes
watchChanges($haikus);
?>
